content/Data visualization and truthful art.md

```diff
@@ -0,0 +1,60 @@
+![Data Visualization](images/data_visualization.jpg)
+
+# Data visualization and truthful art
+An amazing book about data visualization that I can't recommend enough is The Truthful Art by Alberto Cairo.
+
+![Truthful Art](images/truthful_art.png)
+
+In The Truthful Art, Cairo explains the principles of good data visualization. He describes five qualities that should be your foundation when you work with data visualization: truthful, functional, beautiful, insightful, and enlightening. Cairo also gives some great examples of biased and dishonest visualization.
+
+## Data-ink ratio
+
+Before I dive into the "Five Qualities of Great Visualizations," there's another related concept that I want to cover: data-ink ratio, introduced by Edward Tufte in The Visual Display of Quantitative Information.
+
+Tufte defines the data-ink ratio as the amount of data-ink divided by the total ink required to print a graphic. Now, I don't think he asks us to measure the amount of ink laid down on the page. Instead, Tufte suggests we remove those elements that don't add new information to the graphic.
+
+Perhaps one of the best-known examples of data-ink reduction was created by Darkhorse Analytics, an information, visualization, and design company in Edmonton, Alberta, Canada. Darkhorse put together four examples of how the data-ink ratio can be improved. Their main concepts are "data looks better naked" and "remove to improve."
+
+This animation by Darkhorse is super illustrative:
+
+![Data ink](images/data-ink.gif)
+
+## Five Qualities of Great Visualizations
+
+According to Cairo, the five qualities aren't independent of the others. All the different qualities are interrelated, and they interact with all the others.
```

Comment:
`the five qualities aren't independent of the others` I think of the others is redundant here